modules/CmpdReg/src/client/ReparentLotView.html

```diff
@@ -0,0 +1,22 @@
+<script type="text/template" id="ReparentLotView" xmlns="http://www.w3.org/1999/html">
+    <h2 class="form_title bv_title bv_reparentLotTitle"></h2>
+	<div class="bv_dependencyCheckError hide" style="">
+		<h1>Failed to check dependencies.</h1>
+	</div>
+	<div class="bv_reparentLotError hide" style="">
+		<h1>Failed to reparent lot. Please contact your administrator.</h1>
+	</div>
+	<div class="bv_reparentLotSuccess hide" style="">
+		<div class="bv_reparentLotSuccessSummary"></div>
+		<div class="MetaLotViewButtons buttons">
+			<input type=button class="button cregBtn bv_backToCreg small" style="margin-left:20px;float:left;" value="Back Home">
+		</div>
+		<div class= "bv_remainingLotsOnParentLinks"></div>
+	</div>
+	<div class="bv_dependencySummary hide" style=""></div>
+	<div class="MetaLotViewButtons buttons reparentLotButtons">
```

Comment:
I'm a little confused at why these are `MetaLotViewButtons`, and how these differ from `ReparentLotButtons` in `ParentView.inc`, but as long as the code works, I'm alright.